src/TBuffer.h

```diff
@@ -264,6 +264,8 @@ class TBuffer
     void decodeOSC(const QString&);
     void resetColors();
 
+    int MAX_LINKS = 2000;
```

Comment:
THIS IS A BIT SHOUTY - AND CONFUSING - IT SUGGESTS A \#DEFINE CONSTANT.

Given the scope (just within `TBuffer`) why not something like `static const int scmMaxLinks` ...?

Oh, and I kicked the 3rd of the 7 Travis CI build jobs to try again - it had errorred out because of *no output in 10 minutes* at the end...

Reply:
Applied feedback